libchisel/src/repack.rs

```diff
@@ -0,0 +1,124 @@
+use super::{ModuleError, ModuleTranslator};
+use parity_wasm::builder;
+use parity_wasm::elements::*;
+
+pub struct Repack;
+
+impl Repack {
+    pub fn new() -> Self {
+        Repack {}
+    }
+}
+
+impl ModuleTranslator for Repack {
+    fn translate_inplace(&self, module: &mut Module) -> Result<bool, ModuleError> {
+        Err(ModuleError::NotSupported)
+    }
+
+    fn translate(&self, module: &Module) -> Result<Option<Module>, ModuleError> {
+        let module = module.clone();
+        let module = builder::from_module(module).build();
```

Comment:
Are we certain that this drops custom sections?

Reply:
Based on empirical tests it drops everything.

Reply:
everything being all "unknown" sections?

Reply:
It dropped the names section too. Technically it just drops every unparsed section and none of the custom sections are parsed by default by pwasm.